configure.ac

```diff
@@ -4618,12 +4618,26 @@ AC_MSG_CHECKING(LDVERSION)
 LDVERSION='$(VERSION)$(ABIFLAGS)'
 AC_MSG_RESULT($LDVERSION)
 
+# platsubdir must be defined before LIBPL definition
+AC_MSG_CHECKING(for custom platsubdir)
+AC_ARG_WITH(custom-platsubdir,
+    [AS_HELP_STRING([--with-custom-platsubdir=<libdirname>],
```

Comment:
instead of `--with-custom...` just use `--with`. Also this is more commonly called `--with-libdir`

Reply:
the whole point of the PR is that this is distinct from (how Python uses) libdir